src/localization/en.json

```diff
@@ -414,7 +414,7 @@
             "export": "Export",
             "reset": "Reset"
         },
-        "community_default_tab": "Default community home page tab",
+        "community_default_tab": "Default tab for the \"Community Hub\" button",
```

Comment:
Not sure if the "Community Hub" button on app pages is the only link to such pages

Reply:
Oh, I thought this feature specifically targets the "Community Hub" button. Does it work elsewhere?

Reply:
Not that I know of any currently, but the feature is looking for any link like this, it doesn't query this specific element, so it's subject to change

Reply:
ok, I'll revert this change for now.

Reply:
Alright